include/uxr/agent/transport/tcp/TCPServerWindows.hpp

```diff
@@ -39,11 +39,11 @@ class TCPConnectionPlatform : public TCPConnection
 class TCPServer : public TCPServerBase
 {
 public:
-    microxrcedds_agent_DllAPI TCPServer(uint16_t port);
+    microxrcedds_agent_DllAPI TCPServer(uint16_t port, uint16_t discovery_port);
     microxrcedds_agent_DllAPI ~TCPServer() = default;
 
 private:
-    virtual bool init() override;
+    virtual bool init(bool discovery_enabled) override;
```

Comment:
To keep consistency remove `virtual` from derived class overridden methods. If the derived class should not have more derived, consider `final` keyword.

Inconsistency between SerialServers and TCP and UDP servers.